packages/extension-chakra-storefront/src/components/image-gallery/index.jsx

```diff
@@ -99,13 +106,14 @@ const ImageGallery = ({imageGroups = [], selectedVariationAttributes = {}, size,
     const thumbnailImages = thumbnailImageGroup?.images || []
     const loadingStrategy = lazy ? 'lazy' : 'eager'
 
-    const heroImageMaxWidth = styles.heroImage.maxWidth[3] // in px
+    const responsiveMaxWidthValues = Object.values(styles.heroImage.maxWidth['@layer recipes'])
+    const heroImageMaxWidth = useBreakpointValue(responsiveMaxWidthValues) // in px
```

Comment:
Can we simplify this?

```
 const heroImageMaxWidth = useBreakpointValue(styles.heroImage.maxWidth) 
 ```

Reply:
Unfortunately this does not work. Let's just role with it, and we can loop back if we find another alternative. That being said, we might want to ask ourselves if it makes sense to have responsive values in the sized variants?